learning_table_tennis_from_scratch/hysr_one_ball.py

```diff
@@ -26,10 +26,23 @@
 SEGMENT_ID_STEP_FREQUENCY = "hysr_step_frequency"
 
 
+
+def _to_robot_type(robot_type:str)->pam_mujoco.RobotType:
+    if robot_type=="pamy1":
+        return pam_mujoco.RobotType.PAMY1
+    elif robot_type=="pamy2":
+        return pam_mujoco.RobotType.PAMY2
+    else:
+        error = str("hysr configuration robot_type should be either "
+                    "'pamy1' or 'pamy2' (entered value: {})").format(robot_type)
+        raise ValueError(error)
```

Comment:
Python enums allow dictionary-like access, so you could just do `RobotType[robot_type.upper()]` to convert the string.  The only drawback I see, is that you would get a less explicit error message in case of an invalid input.